pychunkedgraph/app/segmentation/v1/routes.py

```diff
@@ -129,7 +136,20 @@ def handle_root(table_id, node_id):
 def handle_roots(table_id):
     int64_as_str = request.args.get("int64_as_str", default=False, type=toboolean)
     root_ids = common.handle_roots(table_id)
-    return jsonify_with_kwargs(root_ids, int64_as_str=int64_as_str)
+    resp = {"root_ids": root_ids}
+
+    arg_as_binary = request.args.get("as_binary", default="", type=str)
+    if arg_as_binary in resp:
```

Comment:
The user passes `as_binary` to specify a single key in the response dict to get its content (whatever it may be) returned as flat binary, correct? In this specific case, the only available key will be `roots_ids`, correct?

Reply:
That's right